build/media-suite_compile.sh

```diff
@@ -1760,12 +1760,12 @@ if [[ $ffmpeg != "no" ]]; then
 
         if [[ $ffmpeg = "both" ]]; then
             _check+=(bin-video/ffmpegSHARED/lib/"libavutil${build_suffix}.dll.a")
-            FFMPEG_OPTS_SHARED+=(--prefix="$LOCALDESTDIR/bin-video/ffmpegSHARED")
+            FFMPEG_OPTS_SHARED+=("--prefix=\"$LOCALDESTDIR/bin-video/ffmpegSHARED\"")
```

Comment:
Doesn't `"--prefix=$LOCALDESTDIR/bin-video/ffmpegSHARED"` work just as well, like above?

Reply:
I would think so too, but at the same time, I'm slightly concerned if we should account for people who might try to redirect $LOCALDESTDIR to a separate folder, or should we just not care about those kinds of cases?

Reply:
Fuck them.

Reply:
Yeah, this should be removed. Literal quotes will break FFmpeg's configure.

Reply:
Removed